sycl/doc/extensions/experimental/sycl_ext_oneapi_matrix/sycl_ext_oneapi_matrix.asciidoc

```diff
@@ -56,37 +56,31 @@ value to determine which of the extension's APIs the implementation supports.
 
 ## Matrix API Versions
 
-While this document presents the core API that unifies Intel AMX, DPAS, and Nvidia Tensor Cores, the implementations support slightly different versions of the API. For this reason, we introduce a new macro, namely `SYCL_EXT_ONEAPI_MATRIX_VERSION`  to distinguish between these different implementations. The goal in the next few months is to get rid of this implementation versioning macro. These are the current values for this macro.
+While this document presents the core API that unifies Intel AMX, Intel XMX, and Nvidia Tensor Cores, the implementations support slightly different versions of the API. For this reason, we introduce a new macro, namely `SYCL_EXT_ONEAPI_MATRIX_VERSION`  to distinguish between these different implementations. The goal in the next few months is to get rid of this implementation versioning macro. These are the current values for this macro.
 
 [frame="none",options="header"]
 |======================
 |Value |Description
-|1     |Initial extension JIT implementation on Intel AMX and DPAS. load, store, mad, fill, piece-wise operations, and the query interface are supported. The old API used for this implementation is detailed in [matrix extension](doc/extensions/deprecated/sycl_ext_oneapi_deprecated_matrix_no_use.asciidoc)
-|2     |JIT implementation on Intel AMX and DPAS. load, store, mad, fill, piece-wise operations, and the query interface are supported 
+|1     |Initial extension JIT implementation on Intel AMX and Intel XMX. load, store, mad, fill, piece-wise operations, and the query interface are supported. The old API used for this implementation is detailed in [matrix extension](../../deprecated/sycl_ext_oneapi_matrix_no_use.asciidoc)
+|2     |JIT implementation on Intel AMX and Intel XMX. load, store, mad, fill, piece-wise operations, and the query interface are supported 
 |3     |Implementation on Nvidia Tensor Cores
 |======================
 
 ## New `joint_matrix` class
-We introduce a new class called `joint_matrix`. The user needs to specify the type of the elements, shape, the matrix use, the memory layout, and the memory scope of the matrix. This results in the following description:
+We introduce a new class called `joint_matrix`. The user needs to specify the memory scope, the type of the elements, the shape, the matrix use, and the memory layout of the matrix. This results in the following description:
```

Comment:
I'm not sure "memory scope" is the correct term here for the Group template argument: Isn't it "group scope"?. In any case it should be made clear that the options here are sub_group or group.
```suggestion
We introduce a new class called `joint_matrix`. The user needs to specify the group scope (either sycl::group or sycl::sub_group), the type of the elements, the shape, the matrix use, and the memory layout of the matrix. This results in the following description:
```

Reply:
memory scope terminology exists in the SYCL spec and it is exactly what we refers to here